src/settings/TransferAccountsSettings.js

```diff
@@ -236,7 +236,7 @@ class TransferAccountsSettings extends React.Component {
             validate={this.validate}
             visibleFields={['accountName', 'desc', 'lastUpdated']}
             columnMapping={{
-              accountName: formatMessage({ id: 'ui-users.transfers.columns.name' }),
+              accountName: formatMessage({ id: 'ui-users.transfers.columns.name' })+'*',
```

Comment:
@JohnC-80 interesting question here: how (if at all) should a required field be labeled in the `<EditableList>` component since it's displayed as a column header rather than as a form-field label?